torch/utils/flop_counter.py

```diff
@@ -153,6 +153,14 @@ def conv_flop(x_shape, w_shape, _bias, _stride, _padding, _dilation, transposed,
     return conv_flop_count(x_shape, w_shape, out_shape, transposed=transposed)
 
 
+@register_flop_formula([aten.add, aten.sub, aten.mul, aten.div, aten.pow, aten.remainder, aten.add_, aten.sub_, aten.mul_, aten.div_, aten.pow_, aten.remainder_])
```

Comment:
is `pow` exactly one ? no, right ? why not just check torch.tag.Pointwise if this isnt exactly right anyway